main.tf

```diff
@@ -42,6 +42,7 @@ module "build" {
   { APP_NAME = var.app_name,
     ENV_TYPE = var.env_type,
     ENV_NAME = var.env_name,
+    WORKSPACE_NAME = split("-", terraform.workspace)[0],
```

Comment:
What is the difference between ENV_NAME, WORKSPACE_NAME and terraform.workspace
Why do we need them all ?